dask/multiprocessing.py

```diff
@@ -148,8 +166,9 @@ def get(dsk, keys, num_workers=None, func_loads=None, func_dumps=None,
     """
     pool = config.get('pool', None)
     if pool is None:
-        pool = multiprocessing.Pool(num_workers,
-                                    initializer=initialize_worker_process)
+        context = get_context()
```

Comment:
If someone uses `forkserver` then does a new server get spawned every time here, or are they reused between calls?

Reply:
Contexts are singleton in Python at the moment.

Reply:
Are you imagine caching contexts, @mrocklin? Can see that being important particularly if we want re-entrant behavior.

Reply:
We currently do this once at import time for dask.distributed.  The context is global.

Reply:
So create all possible contexts at import and then choose between them here? Maybe add them to a dictionary? Other thoughts?

Reply:
Python already has singleton contexts, there's no need to duplicate that behavior.

Reply:
Indeed

```python
In [1]: import multiprocessing

In [2]: multiprocessing.get_context('forkserver') is multiprocessing.get_context('forkserver')
Out[2]: True
```